src/Windows/ViewModels/AboutViewModel.cs

```diff
@@ -8,11 +8,11 @@
 
 namespace AvaloniaCoreRTDemo.Windows.ViewModels
 {
-    internal record SystemDetail(string Key, string Value);
-    
+    internal record SystemDetail(String Key, String Value);
```

Comment:
What's wrong with using C# keywords? I prefer to not introduce needless changes.

Reply:
I reverted this, as you point this change is needless. I made it just cause the whole class use System types instead C# alias. We never define nothing about that.